Jenkinsfile.crossbuild

```diff
@@ -0,0 +1,224 @@
+                            '''
+            
+                            sh '''#!/bin/bash -l +x
+                                security import $MAC_CERT -P $MAC_CERT_PASS -k cliqz -A
+                                security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k cliqz cliqz
+                            '''
+                            withEnv(["MAC_CERT_NAME=$MAC_CERT_NAME"]){
+                                sh "./sign_mac.sh"
+                            }
+                        } finally {
+                            sh '''#!/bin/bash -l -x
+                                security delete-keychain cliqz
+                                security list-keychains -s login.keychain
+                                security default-keychain -s login.keychain
+                                true
+                            '''
+                        }
+                    }
+                }
+                stage('Stash signed artifacts'){
+                    stash includes: 'obj/dist/*de.mac.dmg', name: 'signeddedmg'
+                    stash includes: 'obj/dist/*US.mac.dmg', name: 'signedusdmg'
+                     sh """#!/bin/bash -l
+                    set -e
+                    set -x
+                    rm -rf obj/dist/*mac.dmg
+                """
+                }
+            }
+            catch(all) {
+                jobStatus = 'FAIL'
+                print "Something Failed. Check the above logs."
+                currentBuild.result = 'FAILURE'
+            }
+        }
+    }
+}
+image.inside("-w /opt/browser-f"){
+    stage('Unstash signed artifacts'){
+        unstash 'signeddedmg'
+        unstash 'signedusdmg'
+    }
+    stage('upload') {
+        withEnv(['CQZ_CERT_DB_PATH=/home/jenkins/certs', "CQZ_BUILD_DE_LOCALIZATION=$CQZ_BUILD_DE_LOCALIZATION"]) {
+            try {
+                //expose certs
+                withCredentials([
+                    [$class: 'FileBinding', credentialsId: MAR_CERT_CREDENTIAL_ID, variable: 'MAR_CERT'],
+                    [$class: 'StringBinding', credentialsId: MAR_CERT_PASS_CREDENTIAL_ID, variable: 'MAR_CERT_PASS']]) {
+
+                    sh '''#!/bin/bash -x
+                        mkdir $CQZ_CERT_DB_PATH
+                        certutil -N -d $CQZ_CERT_DB_PATH --empty-password
+                        set +x
+                        pk12util -i $MAR_CERT -W $MAR_CERT_PASS -d $CQZ_CERT_DB_PATH
+                        certutil -L -d $CQZ_CERT_DB_PATH
+                    '''
+                }
+
+                withCredentials([[
+                    $class: 'UsernamePasswordMultiBinding',
+                    credentialsId: CQZ_AWS_CREDENTIAL_ID,
+                    passwordVariable: 'AWS_SECRET_ACCESS_KEY',
+                    usernameVariable: 'AWS_ACCESS_KEY_ID']]) {
+
+                    sh """#!/bin/bash -l 
+                        set -x
+                        ./magic_upload_files_crossbuild.sh
```

Comment:
Let's try to use same file as we have for normal build. You can use OSX_CROSS_BUILD env. variable to specify proper path (dist/bin or /opt/) in original "magic_upload_files.sh" script